src/Illuminate/Support/Str.php

```diff
@@ -794,4 +795,38 @@ public static function createUuidsNormally()
     {
         static::$uuidFactory = null;
     }
+
+    /**
+     * Splits the string by delimiter into a collection of stringable objects.
+     *
+     * @param  string  $string
+     * @param  string  $delimiter
+     * @param  int|null  $limit
+     *
+     * @return \Illuminate\Support\Collection
+     */
+    public static function split($string, $delimiter = ',', $limit = null)
+    {
+        $pieces = explode(...array_filter([$delimiter, $string, $limit]));
```

Comment:
You should allow `$delimiter` to be falsey - might be best to avoid `array_filter()`